index.js

```diff
@@ -91,15 +90,15 @@ exports.handler = async event => {
             bytes: total,
             seconds: totalSeconds,
             percent: totalPercent,
-            percentAds: ads,
+            durations: arr.durations(),
+            types: arr.types,
```

Comment:
makes sense to me - passing through more data, but not so much that we're changing the data flow significantly, and yeah, kick the can to the analytics lambda makes sense.